.changelog/unreleased/bug-fixes/ibc-relayer-cli/2429-fix-oldest-sequence-timeout.md

```diff
@@ -0,0 +1,2 @@
+- Updates sequence_histories when a timeout occurs
```

Comment:
```suggestion
- Ensure `oldest_sequence` and `oldest_timestamp` metrics are updated when a timeout occurs
```

Reply:
I am suggesting the change above because changelog entries should be written with users of the relayer in mind, to the extent that it is possible of course.

Reply:
Done !